src/services/refactors/helpers.ts

```diff
@@ -80,6 +80,10 @@ export function addTargetFileImports(
         if (checker.isUnknownSymbol(targetSymbol)) {
             importAdder.addVerbatimImport(Debug.checkDefined(declaration ?? findAncestor(symbol.declarations?.[0], isAnyImportOrRequireStatement)));
         }
+        else if (targetSymbol.parent === undefined) {
+            Debug.assert(declaration !== undefined, "expected module symbol to have a declaration");
+            importAdder.addImportForModuleSymbol(symbol, isValidTypeOnlyUseSite, declaration);
+        }

```

Comment:
Is there a reason why we can't use `isExternalModuleSymbol()` as a check?

Reply:
Do you mean in the if statement or the debug statement? The debug is needed since the intent of the importAdder function is to copy from an existing declaration

Reply:
For the if statement, namespaces are specifically _not_ external module symbols, so `isExternalModuleSymbol` wouldn't work if trying to export something that was defined as a namespace (`React` for example)